packages/venia-ui/lib/util/apolloCache.js

```diff
@@ -42,7 +43,8 @@ export const cacheKeyFromType = object => {
             return object.url_key
                 ? `${MagentoGraphQLTypes.ProductInterface}:${object.url_key}`
                 : defaultDataIdFromObject(object);
-
+        case MagentoGraphQLTypes.SelectedConfigurableOption:
```

Comment:
I think a comment here explaining why we have to do this is warranted.

Reply:
Also looks like we are getting `value_id` from the `getCartDetails` query. I'm not sure that this is just some magical thing we have. We may want to make sure we add it to the product details query within the `CartPage/` components

Reply:
Man, good catch! I had assumed Apollo automatically injected this key, but I was mistaken. The bug appeared to be fixed because we weren't caching any options, because `object.value_id` was `undefined` 🤔

I've now fixed this up in a187f13  so it uses that field if available, and explicitly won't cache if its left out.